spec/spec_helper.rb

```diff
@@ -14,23 +14,30 @@ def define_presenter(name = "ShowPresenter", &block)
     presenter_class
   end
 
-  def render(source, locals = {}, presenter_class = nil, &block)
-    if presenter_class.nil?
-      unless defined?(ShowPresenter)
-        define_presenter("ShowPresenter")
-      end
+  def render(source, options = {}, &block)
```

Comment:
Maybe adding the `presenter` parameter as the third positional argument would be better – that way we don't need to change the specs so much, and it makes refactoring other branches a lot simpler.